Doc/Manual/Guile.html

```diff
@@ -30,6 +30,7 @@
 <li><a href="#Guile_nn14">Smobs</a>
 <li><a href="#Guile_nn15">Garbage Collection</a>
 </ul>
+<li><a href="#Guile_nn12">Native Guile pointers</a>
```

Comment:
There's already a `#Guile_nn12` in this file, so adding a second isn't right.

I'm not sure what we normally do when inserting a section - renumbering the existing anchors seems unhelpful, as it'll break external bookmarks.  @wsfulton What's the preferred way to handle this?